### DISCUSSION TITLE
Facing Issues with the Attu GUI · milvus-io/milvus

### DISCUSSION THREAD

**Discussion by mohit19906** [milvus-io/milvus#26474] [Q&A and General discussion]
We have followed step by step installation to setup Attu GUI from following link - https://milvus.io/docs/v2.1.x/attu_install-docker.md
It was working for couple of days but then we started getting issues.
I am facing an issue while using Attu, I am using docker-image to run the Attu at the localhost, but somehow after connecting the collections keep on loading and after few seconds flashes following error,
Error: 14 UNAVAILABLE: Stream refused by server
For me milvus is connected as I checked it from the script.

I have tried multiple things still the problem persists. Please suggest a solution as this tool is great to visualize the data stored and collections.
Adding top contributors if anyone can help:
[@yhmo](https://github.com/yhmo)
[@xiaofan-luan](https://github.com/xiaofan-luan)

**Comment by yhmo** [comment]
"Error: 14 UNAVAILABLE: Stream refused by server" is an error from grpc-node.
According to this thread: https://github.com/grpc/grpc-node/issues/1969, "The server may refuse to serve new requests if it is already handling too many other requests. This is normal behavior. Your client code will need to be able to recover from that error."

@shanghaikid  Any idea?

**Comment by shanghaikid** [comment]
How many collections do you have?

**Reply by mohit19906** [reply]
We had around 20 Collections. Out of those, we have deleted some collections and now left with 7 Collections. We are intermittently facing issue with making connection to our Milvus server on GUI. We can see in the Network tab that the connection was cancelled. The issue is happening only with Attu. We are able to make connection with Milvus through code and even able to perform actions like drop, create, describe collections.

**Reply by mohit19906** [reply]
@shanghaikid can you please help. We are still facing this issue.

**Reply by shanghaikid** [reply]
Which version is your attu? Maybe you can try to upgrade attu to the latest version.
